README.md

```diff
@@ -409,4 +411,4 @@ queued-exporters:
 [gitter-image]: https://badges.gitter.im/census-instrumentation/lobby.svg
 [gitter-url]: https://gitter.im/census-instrumentation/lobby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge
 [codecov-image]: https://codecov.io/gh/census-instrumentation/opencensus-service/branch/master/graph/badge.svg
-[codecov-url]: https://codecov.io/gh/census-instrumentation/opencensus-service/branch/master/
\ No newline at end of file
+[codecov-url]: https://codecov.io/gh/census-instrumentation/opencensus-service/branch/master/
```

Comment:
Don't understand what changed here?

Reply:
This is some non-intentional whitespace change...

Reply:
(lesson do not edit files via gh web page, I can't find what changed)

Reply:
I think this may be some strange character. Can you please clone this and see what changed?

Reply:
It adds an empty new-line at the end even if I remove it.

Reply:
ok. thanks